users/vieting/experiments/switchboard/ctc/feat/transducer_system_v2.py

```diff
@@ -923,10 +923,8 @@ def run_test_recog_step(
             )
 
     def run_align_step(self, align_args: Optional[Dict] = None) -> None:
-        for train_exp_name in self.returnn_configs.keys():
-            for al_c in self.align_corpora:
-                self._nn_alignment(
-                    train_exp_name,
-                    align_corpus_key=al_c,
-                    **(align_args or {}),
-                )
+        for al_c in self.align_corpora:
+            self._nn_alignment(
+                align_corpus_key=al_c,
+                **(align_args or {}),
+            )
```

Comment:
This is alright, too. But I would still prefer to keep the adjustment of line 864. If you get a key error, then just add this key (`"train"` if it does not yet exist). Otherwise, it's unclear which model created the alignment.